examples/acados_python/pendulum_example/soft_constraint_example.py

```diff
@@ -0,0 +1,237 @@
+#
```

Comment:
Mainly I was moving it there to have the `export_pendulum_ode_model` available  without duplicating it.
Maybe we can just rename the folder `pendulum_example` to `getting_started`.
This would make clear where the user should look.
What do you think?

Reply:
yes, sounds good.